tests/lib/rules/forbid-component-props.js

```diff
@@ -81,6 +81,17 @@ ruleTester.run('forbid-component-props', rule, {
     ].join('\n'),
     options: [{forbid: ['style', 'foo']}],
     parserOptions: parserOptions
+  }, {
+    code: [
+      'var First = React.createClass({',
```

Comment:
no - it's for "stateless functional component" - ie, a function that takes props and returns JSX